Automattic/wp-calypso
# Hosting Command Palette: Add command to navigate to Jetpack dashboard

Related to https://github.com/Automattic/dotcom-forge/issues/4859

## Proposed Changes

This PR adds a command to navigate to the Jetpack dashboard in `wp-admin`. This command is available either for Atomic sites or for self-hosted sites. 

<img width="1122" alt="Screenshot 2023-12-21 at 12 09 17 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/0fa5d02f-b505-4187-8c36-cf534353127d">

<img width="1019" alt="Screenshot 2023-12-21 at 12 00 10 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/bbeedde5-89ba-4ea1-bb21-d0dde51c6e82">

## Testing Instructions

* Navigate to Calypso Live Link
* Press `cmd + k` on macOS or `ctrl + k` on Windows to open the command palette
* Search for the `Open Jetpack settings` command
* Select the command
* Confirm that you can only see Atomic (including staging) or self-hosted sites in the list
* Select an Atomic site to navigate to and trigger the command
* Confirm that you are brought to Jetpack dashboard when executing the command: `wp-admin/admin.php?page=jetpack#/dashboard`

## Pre-merge Checklist
